Description: Drop the CHECK_GLIBC call. We're not compiling Firefox to
 run on a different glibc version
Author: Chris Coulson <chris.coulson@canonical.com>
Forwarded: not-needed

Index: firefox-53.0~b1+build1/config/config.mk
===================================================================
--- firefox-53.0~b1+build1.orig/config/config.mk
+++ firefox-53.0~b1+build1/config/config.mk
@@ -579,7 +579,7 @@ endef
 
 ifneq (,$(MOZ_LIBSTDCXX_TARGET_VERSION)$(MOZ_LIBSTDCXX_HOST_VERSION))
 CHECK_STDCXX = $(call CHECK_SYMBOLS,$(1),GLIBCXX,libstdc++,v[1] > 3 || (v[1] == 3 && v[2] == 4 && v[3] > 16))
-CHECK_GLIBC = $(call CHECK_SYMBOLS,$(1),GLIBC,libc,v[1] > 2 || (v[1] == 2 && v[2] > 12))
+#CHECK_GLIBC = $(call CHECK_SYMBOLS,$(1),GLIBC,libc,v[1] > 2 || (v[1] == 2 && v[2] > 12))
 endif
 
 ifeq (,$(filter $(OS_TARGET),WINNT Darwin))
